### PR TITLE
Parity violating contributions to molecular energy.

### PR DESCRIPTION
**Pull request by juanjoaucar**

### CODE REVIEW

File: pyscf/pv/test/test_energy-pv.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    mf.kernel()
+
+def tearDownModule():
+    global mf
+    mol.stdout.close()
+    del mol, mf
+
+class KnownValues(unittest.TestCase):
+    def test_pv(self):
+#       Reference results (for PV) are obtained from DIRAC24
```
Comment (MatthewRHermes) [inline pyscf/pv/test/test_energy-pv.py:59]: I am not very familiar with this enterprise, but is -2.745821e-21 sufficiently far from zero that we can be sure that finite-precision rounding errors and SCF tolerances are not controlling it? Is there a way to generate a fantasy system with a much larger parity violation, perhaps by manipulating fundamental constants such as the speed of light?

**Comment by sunqm** [comment]
From PR https://github.com/pyscf/pyscf/pull/2459

File: pyscf/pv/test/test_energy-pv.py (hunk 1 above)
Comment (sunqm) [reply pyscf/pv/test/test_energy-pv.py:59]: Maybe test a relative value?

Comment (juanjoaucar) [reply pyscf/pv/test/test_energy-pv.py]: Thanks for the suggestions.
The problem was not the finite-precision rounding errors and SCF tolerances but naive mistakes I've made while changing name variables. It should finish well now.
The tiny results are in part due to the Fermi Constant factor. They are expected to be like that.

**Review by sunqm — APPROVED** [review]

**Pull request merged**